src/Algebra/Module/Morphism/Linear/Properties.agda

```diff
@@ -0,0 +1,172 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of linear maps.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra                   using (CommutativeRing)
+open import Algebra.Module            using (Module)
+open import Level                     using (Level; _⊔_)
+
+module Algebra.Module.Morphism.Linear.Properties
+  {r ℓr m ℓm : Level}
+  {ring      : CommutativeRing r ℓr}
+  (modA modB  : Module ring m ℓm)
+  where
+
+import Algebra.Module.Properties          as Properties
+import Function.Definitions
+import Relation.Binary.Reasoning.Setoid   as Reasoning
+import Algebra.Module.Morphism.Structures as MorphismStructures
+
+open import Agda.Builtin.Sigma
+open import Axiom.DoubleNegationElimination
+open import Data.List
+open import Data.Product
+  using (Σ; _,_; ∃; Σ-syntax; ∃-syntax; _×_)
+open import Relation.Nullary          using (¬_)
+open import Relation.Nullary.Negation using (contraposition)
+
+open MorphismStructures.ModuleMorphisms modA modB
+module A = Module modA
+open A using () renaming (Carrierᴹ to A)
+open Properties A.leftSemimodule
```

Comment:
Again instead of the renaming, we can overload the module name `A` and write:
```agda
module A = Properties A.leftSemimodule

.... A.non-zeroʳ ....
```

Reply:
When I try this, I'm told:

```
Duplicate definition of module A. Previous definition of (record)
module A at
```

Reply:
Since I've already got modules "A" and "B", I've chosen "PA" and "PB" for the _properties_ of modules A and B, respectively.

```agda
module A = Module modA
open A using () renaming (Carrierᴹ to A)
module B = Module modB
open B using () renaming (Carrierᴹ to B)
module PA = ModuleProperties modA
module PB = ModuleProperties modB
```